models/activity/src/index.ts

```diff
@@ -117,6 +119,9 @@ export class TActivityMessage extends TAttachedDoc implements ActivityMessage {
   @Index(IndexKind.Indexed)
     lastReply?: Timestamp
 
+  @Prop(TypeBoolean(), core.string.Boolean)
```

Comment:
We need new member notification instead of hide.

Reply:
Ok, now we only send `notification` about `mention`
And if text of `mention` content  was changed, then we send a new `notification`